Library/Homebrew/version.rb

```diff
@@ -37,6 +61,8 @@ def initialize(value = nil)
     end
 
     def <=>(other)
+      return unless other = Token.from(other)
```

Comment:
Should this be returning `nil` or `-1`? It feels like this should maybe be part of the else case?

Reply:
`nil` represents that the two types are not comparable. In this case, if `other` isn't a valid token (and cannot be converted to one).

This line will also do a conversion from int/string to a token before hitting the `case` checks.

Reply:
Works for me, thanks @Bo98!